src/shared/ListRepo/ReposTableTeam/ReposTableTeam.tsx

```diff
@@ -0,0 +1,286 @@
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import type { SortingState } from '@tanstack/react-table'
+import cs from 'classnames'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import { useContext, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import {
+  OrderingDirection,
+  Repository,
+  TeamOrdering,
+  useReposTeam,
+} from 'services/repos'
+import { ActiveContext } from 'shared/context'
+import { formatTimeToNow } from 'shared/utils/dates'
+import Button from 'ui/Button'
+import Icon from 'ui/Icon'
+
+import InactiveRepo from '../InactiveRepo'
+import { repoDisplayOptions } from '../ListRepo'
+import NoReposBlock from '../NoReposBlock'
+import 'ui/Table/Table.css'
+import RepoTitleLink from '../RepoTitleLink'
+
+export function getSortingOption(
+  sorting: Array<{ id: string; desc: boolean }>
+) {
+  const state = sorting.at(0)
+
+  if (state) {
+    const direction = state?.desc
+      ? OrderingDirection.DESC
+      : OrderingDirection.ASC
+
+    let ordering
+    if (state.id === 'name') {
+      ordering = TeamOrdering.NAME
+    }
+
+    if (state.id === 'latestCommitAt') {
+      ordering = TeamOrdering.COMMIT_DATE
+    }
+
+    return { direction, ordering }
+  }
+
+  return undefined
+}
+
+const columnHelper = createColumnHelper<Repository>()
+
+interface ReposTableTeamProps {
+  searchValue: string
+}
+
+const getColumns = ({
+  inactive,
+  isCurrentUserPartOfOrg,
+}: {
+  inactive: boolean
+  isCurrentUserPartOfOrg: boolean
+}) => {
+  if (inactive) {
+    return [
+      columnHelper.accessor('name', {
+        header: 'Name',
+        id: 'name',
+        cell: (info) => {
+          const repo = info.row.original
+          return (
+            <RepoTitleLink
+              repo={repo}
+              showRepoOwner={!!repo?.author?.username}
+              pageName={!!repo?.active ? 'repo' : 'new'}
+              disabledLink={!isCurrentUserPartOfOrg && !repo?.active}
+            />
+          )
+        },
+      }),
+      columnHelper.accessor('lines', {
+        header: '',
+        id: 'lines',
+        cell: (info) => {
+          const repo = info.row.original
+          return (
+            <InactiveRepo
+              owner={repo?.author?.username ?? ''}
+              repoName={repo?.name}
+              isCurrentUserPartOfOrg={isCurrentUserPartOfOrg}
+              isActive={!!repo?.active}
+            />
+          )
+        },
+      }),
+    ]
+  }
+  return [
+    columnHelper.accessor('name', {
```

Comment:
Consider splitting this into its own variable so you only have to declare it once. Then you can do something like 
```
const nameColumn = columnHelper.accessor('name', {
  header: 'Name',
  id: 'name',
  ...
```
and then

return `[nameColumn, columnHelper.accessor(...), etc]`.

 This will help reduce repeated code in the same file and help with readability.